src/Adapters/Email/MailAdapter.js

```diff
@@ -0,0 +1,6 @@
+export class MailAdapter {
+  sendVerificationEmail(options) {}
+  sendMail(options) {}
```

Comment:
I'm no longer sure we want to mandate the adapter include a `sendMail` function, some adapters may want to verify via text message/Facebook message/something else instead. It would be a bit hacky if someone did that, but I'd rather expose the smallest possible interface, just to leave that option open in the future.